cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/GcpNetworkConnector.java

```diff
@@ -149,21 +149,21 @@ public NetworkCidr getNetworkCidr(Network network, CloudCredential credential) {
         String projectId = GcpStackUtil.getProjectId(credential);
         Subnetwork subnet = null;
         try {
-            SubnetworkList ownProjectSubnets = compute.subnetworks().list(projectId, region).execute();
-            Set<Subnetwork> collect = new HashSet<>();
-            if (ownProjectSubnets.getItems() != null) {
-                collect = ownProjectSubnets.getItems()
-                        .stream()
-                        .filter(e -> e.getName().equals(subnetId) || e.getId().toString().equals(subnetId))
-                        .collect(Collectors.toSet());
-            }
-            if (collect.isEmpty() && !Strings.isNullOrEmpty(sharedProjectId)) {
-                subnet = compute.subnetworks().get(sharedProjectId, region, subnetId).execute();
-            } else {
+            if (Strings.isNullOrEmpty(sharedProjectId)) {
```

Comment:
a log would be good, the subnet searched by sharedProjectId or not

Reply:
done